ios/App/App/capacitor.config.json

```diff
@@ -9,6 +9,7 @@
   },
   "plugins": {
     "Keyboard": {
+      "resize": "none",
```

Comment:
@faboweb i think this file might be overridden during the bitrise build. can we confirm somehow?

Reply:
if following the same steps from bitrise, will it get overridden?